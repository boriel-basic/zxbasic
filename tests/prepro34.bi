' a Single test with continued lines

LET a = _ REM should be continued
 5

