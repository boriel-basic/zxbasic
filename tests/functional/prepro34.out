#line 1 "prepro34.bi"


LET a = _ 
 5

